scss/_patterns_search-and-filter.scss

```diff
@@ -113,12 +114,9 @@
       flex-grow: 1;
       margin-bottom: 0;
       margin-left: -$sph-inner--small; // compensate for the left padding of the container
-      margin-right: -$sp-x-large; // compensate for the space reserved for counter
-      min-height: 2.25rem;
+      margin-right: calc(calc(#{$search-filter-actions-width} + #{$focus-outline-offset}) * -1); // compensate for the space reserved for counter
```

Comment:
Does it need to be double `calc`? Can't it be in one?

Reply:
No, it needs to be double, otherwise it performs the multiplication first, and I need that to happen _last_, so that inverts the value of the inner calc and makes it negative - unless there's another way to do that I don't know about?

Reply:
you can just do calc(-val1 - val2)